lib/u2f/register_response.rb

```diff
@@ -20,6 +21,10 @@ def self.load_from_json(json)
         raise RegistrationError, code: data['errorCode']
       end
 
+      if !data.key?('clientData') || !data.key?('registrationData')
+        raise RegistrationError, code: BAD_REQUEST
```

Comment:
Surprisingly that is not my use case. Additionally, the Rails / browser mind set in this library is a very bad idea.